src/Command/CreateSiteCommand.php

```diff
@@ -107,7 +107,16 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         $confirmation = true;
 
-        if (!$input->getOption('no-confirmation')) {
+        if (null !== $input->getOption('no-confirmation')) {
+            $input->setOption('no-interaction', true === $input->getOption('no-confirmation'));
+
+            @trigger_error(
+                'The "no-confirmation" option is deprecated since sonata-project/block-bundle 3.x and will be removed in version 4.0.',
```

Comment:
```suggestion
                'The "no-confirmation" option is deprecated since sonata-project/page-bundle 3.x and will be removed in version 4.0.',
```

Reply:
good catch